src/Client.php

```diff
@@ -402,7 +402,7 @@ public function setPatchType($patchType = "strategic")
 	 */
 	public function sendRequest($method, $uri, $query = [], $body = null, $namespace = true, $apiVersion = null, array $requestOptions = [])
 	{
-		$baseUri = $apiVersion ? 'apis/' . $apiVersion : 'api/' . $this->apiVersion;
+		$baseUri = $apiVersion ? 'api/' . $apiVersion : 'api/' . $this->apiVersion;
```

Comment:
this looked like a typo as well? Not sure if `apis/` is ever the correct endpoint.

Reply:
ah, whoops! i'll change it back